ingenialink/ethercat/servo.py

```diff
@@ -320,6 +323,49 @@ def set_pdo_watchdog_time(self, timeout: float) -> None:
             self.ETHERCAT_PDO_WATCHDOG, self.SECONDS_TO_MS_CONVERSION_FACTOR * timeout
         )
 
+    def read_eeprom(
+        self,
+        address: int,
+        length: int = DEFAULT_EEPROM_READ_BYTES_LENGTH,
```

Comment:
Should we check that length % 2 == 0?

Reply:
It does not need to be a multiple of 2. Check the [test](https://github.com/ingeniamc/ingenialink-python/pull/447/files#diff-7e3660da5a421071fe4996b2da15a790b82f68bb1036f934ca9e8022e3692f66R9).